src/registrar/admin.py

```diff
@@ -187,43 +200,48 @@ class MyUserAdmin(BaseUserAdmin):
         "email",
         "Permissions",
         "is_active",
-        "is_staff",
-        "is_superuser",
+        "groups",
         "Important dates",
         "last_login",
         "date_joined",
     ]
 
     def get_list_display(self, request):
-        if not request.user.is_superuser:
-            # Customize the list display for staff users
-            return (
-                "email",
-                "first_name",
-                "last_name",
-                "is_staff",
-                "is_superuser",
-                "status",
-            )
-
-        # Use the default list display for non-staff users
-        return super().get_list_display(request)
+        # The full_access_permission perm will load onto the full_access_group
+        # which is equivalent to superuser. The other group we use to manage
+        # perms is cisa_analysts_group. cisa_analysts_group will never contain
+        # full_access_permission
+        if request.user.has_perm("registrar.full_access_permission"):
+            # Use the default list display for all access users
+            return super().get_list_display(request)
+
+        # Customize the list display for analysts
+        return (
+            "email",
+            "first_name",
+            "last_name",
+            "group",
+            "status",
+        )
```

Comment:
This seems like something we could store in an external variable somewhere, i.e. "analyst_list_display"